<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>217</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>453</width>
    <height>217</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>453</width>
    <height>217</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BCE++</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblMessage_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>411</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMessage_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>411</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEquation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>411</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtResult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
    <property name="centerOnScroll">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdBalance">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdLoadFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdExit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
